### ISSUE TITLE
Methylphenidate  dependence

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6538]
**Preferred term label:** Methylphenidate  dependence

**Synonyms**
Methylphenidate  addiction 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Stimulants dependence (to be created)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
